src/helpers/loadOrCreateStakePrizePool.ts

```diff
@@ -0,0 +1,42 @@
+import { log, Address } from '@graphprotocol/graph-ts'
+
+import {
+  StakePrizePool,
+} from '../../generated/schema'
+
+import {
+  StakePrizePool as StakePrizePoolContract,
+} from '../../generated/templates/StakePrizePool/StakePrizePool'
+
+import { loadOrCreatePrizePool } from './loadOrCreatePrizePool'
+
+
+export function loadOrCreateStakePrizePool(
+  prizePool: Address
+): StakePrizePool {
+  let _stakePrizePool = StakePrizePool.load(prizePool.toHex())
+  if (!_stakePrizePool) {
+    _stakePrizePool = new StakePrizePool(prizePool.toHex())
+
+    const _prizePool = loadOrCreatePrizePool(prizePool)
+    _prizePool.prizePoolType = 'Stake'
+
+    _prizePool.stakePrizePool = _stakePrizePool.id
+    
+    _prizePool.save()
+
+    const _boundskatePrizePool = StakePrizePoolContract.bind(prizePool)
```

Comment:
Hmm...you're handling the StakePrizePoolInitialized event, so I don't think this code is needed.  The initialized event sets up the `stakeToken`